xml/art_sle_ha_install_quick.xml

```diff
@@ -675,22 +704,22 @@ Sep 22 17:01:00 &node1; sbd: [13412]: info: Received command test from &node1;</
    </step>
   </procedure>
 
-  <para><remark>taroth 2016-07-25: replace with Hawk2!</remark>
-   Check the cluster status with <command>crm&nbsp;status</command>. If
-   you have successfully added a second node, the output will be similar to
-   the following:
+  <para>
+   Check the cluster status in &hawk2;. Under <menuchoice>
+    <guimenu>Status</guimenu>
+    <guimenu>Nodes</guimenu>
+   </menuchoice> you should see two nodes all with a green status (see
+   <xref linkend="fig.ha.inst.quick.two-node-cluster"/>).
   </para>
 
-<screen>&prompt.root;<command>crm</command> status
-Stack: corosync
-Current DC: &node1; (version 1.1.15-17.1-e174ec8) - partition with quorum
-Last updated: Fri Jul 15 12:33:08 2016
-Last change: Fri Jul 15 12:33:03 2016 by hacluster via crm_attribute on &node1;
-
-2 Nodes configured
-0 Resources configured
-
-Online: [ &node1; &node2; ]</screen>
+  <figure xml:id="fig.ha.inst.quick.two-node-cluster">
+   <title>Status of a Two-Node Cluster</title>
```

Comment:
s/a/the (I would prefer the definite article here because he have a specific setup with alice and bob - no arbitrary two-node cluster). Let's see what the language proofreader makes out of it. ;)